syntax/gleam.vim

```diff
@@ -50,5 +83,5 @@ highlight link gleamString String
 highlight link gleamStringModifier Special
 
 " Attribute
-syntax match gleamAttribute "#[a-z][a-z_]*"
+syntax match gleamAttribute "@\(external\|deprecated\|target\)"
```

Comment:
Let's make this support any snake case word

Reply:
Done.